### DISCUSSION TITLE
Citekeys Exporting in All-Caps

### DISCUSSION THREAD

**Discussion by thejamesheard** [retorquere/zotero-better-bibtex#2045] [Q&A] [answered]
I use Better BibTex in Scrivener through [zotpick](https://github.com/davepwsmith/zotpick-applescript) and after updating Better BibTex yesterday the citekeys started exporting in all-caps. 

So, what used to look like this: { | Ellis, 1994 | | | zu:6846605:9NGL3Y4G }

Now looks like this: { | ELLIS, 1994 | | | ZU:6846605:9NGL3Y4G }

The only thing I can think of is that yesterday's update is causing the problem, however, a colleague of mine updated without any problems. Any ideas?

**Comment by retorquere** [comment, marked as answer]
I'll need a support log generated after you reproduce this to be able to tell. I haven't changed anything related to citekey generation. Does the same happen if you paste the key elsewhere? What platform are you on?

**Reply by thejamesheard** [reply]
Well, this is embarrassing... Capslock was on. Sort of.

I'm using a MacBook Pro with an external keyboard and, even though capslock was _off_ on it (i.e. I could type and it appeared to be off), capslock was turned _on_ on the MacBook. The zotpick plugin "types" out the citekey into Scrivener rather than copying and pasting and, I guess, uses the MacBook's capslock state rather than that of the external keyboard.

**Reply by retorquere** [reply]
Good that you got it resolved.
